parity-fix/doc/FIX.md

```diff
@@ -0,0 +1,210 @@
+FIX
+===
+
+FIX is an alternative order entry protocol for Parity.
+
+
+Architecture
+------------
+
+Parity uses FIX 4.4. See below for the supported tags.
+
+
+### Message Header
+
+All messages start with the message header.
+
+Tag                  | Req'd | Description
+---------------------|-------|-----------------------------------
+BeginString(8)       |   Y   | FIX.4.4
+BodyLength(9)        |   Y   |
+MsgType(35)          |   Y   |
+SenderCompID(49)     |   Y   |
+TargetCompID(56)     |   Y   |
+MsgSeqNum(34)        |   Y   |
+PossDupFlag(43)      |   N   |
+PossResend(97)       |   N   |
+SendingTime(52)      |   Y   |
+OrigSendingTime(122) |   N   |
+
+
+### Message Trailer
+
+All messages end with the message trailer.
+
+Tag          | Req'd | Description
+-------------|-------|------------
+CheckSum(10) |   Y   |
+
+
+Inbound Messages
+----------------
+
+Inbound messages are sent by the client application to the trading system.
+
+
+### Enter Order
+
+Enter an order with a New Order Single message.
+
+Tag               | Req'd | Description
+------------------|-------|-----------------------------------
+_Message Header_  |   Y   | MsgType(35) = D (New Order Single)
+ClOrdID(11)       |   Y   | Unique identifier for the order
+Symbol(55)        |   Y   |
+Side(54)          |   Y   | See below
+TransactTime(60)  |   Y   |
+OrderQty(38)      |   Y   |
+OrdType(40)       |   Y   | See below
```

Comment:
Do you envision the FIX gateway maintaining the account information for the market participant? The trading system itself has no concept of an account.